dingo/pipe/utils.py

```diff
@@ -3,6 +3,6 @@ def _strip_unwanted_submission_keys(job):
     job.universe = None
     extra_lines = []
     for line in job.extra_lines:
-        if not line.startswith("priority") and not line.startswith("accounting_group"):
+        if not line.startswith("priority") and not line.startswith("accounting_group") and not line.startswith("ENV"):
```

Comment:
Would it work to make this more specific?

```suggestion
        if not line.startswith("priority") and not line.startswith("accounting_group") and not line.startswith("ENV GET HTGETTOKENOPTS"):
```

Reply:
Yes, it works, I just tested it.